### ISSUE TITLE
Task object

### ISSUE THREAD

**Issue by ArlindKadra** [openml/openml-python#506]
#### Description
The task object contains the information regarding estimation parameters twice. Doing a small refactor would remove the duplicate information.

#### Steps/Code to Reproduce
```
from pprint import pprint
import openml

task_id = 2
task = openml.tasks.get_task(task_id)
pprint(vars(task))
```
#### Result
```

{'class_labels': ['1', '2', '3', '4', '5', 'U'],
 'cost_matrix': None,
 'dataset_id': 1,
 'estimation_parameters': {'number_folds': '2',
                           'number_repeats': '5',
                           'percentage': '',
                           'stratified_sampling': 'true'},
 'estimation_procedure': {'data_splits_url': 'https://test.openml.org/api_splits/get/2/Task_2_splits.arff',
                          'parameters': {'number_folds': '2',
                                         'number_repeats': '5',
                                         'percentage': '',
                                         'stratified_sampling': 'true'},
                          'type': 'crossvalidation'},
 'evaluation_measure': None,
 'split': None,
 'target_name': 'class',
 'task_id': 2,
 'task_type': 'Supervised Classification',
 'task_type_id': 1}
```

**Comment by PGijsbers** [comment]
I think this is outdated. Right now it looks like the actual data is not stored twice, but the same dictionary is referenced twice ([e.g. here](https://github.com/openml/openml-python/blob/develop/openml/tasks/task.py#L105)).

If it is just regarding the string representations I think this is duplicate with #653 or #75.

Feel free to re-open if I misunderstood.

**Comment by ArlindKadra** [comment]
Referenced also in #656, addressed in #607
